app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils desktop
+
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+COMMON_DEPEND="
```

Comment:
No `DEPEND`, but you use `COMMON_DEPEND` for some reason? If there is nothing built here, just make this whole block an `RDEPEND` instead.

Reply:
Just included this change.

Happy New Year!